### ISSUE TITLE
Running built .exe file without cmd or powershell

### ISSUE THREAD

**Issue by something-zero** [Prepouce/CoercedPotato#3]
Hello friends.
Thank you for this great and useful tool
my problem is when i want run your tool with using cmd, works good and correct.
i changed a few your codes for remove arguments.
![image](https://github.com/Prepouce/CoercedPotato/assets/149296852/59000acb-69c8-4046-ac34-74ea9db47e3d)

when i run your tool with double-click(run as administrator) show me this errors:
![image](https://github.com/Prepouce/CoercedPotato/assets/149296852/ada80b3d-987d-4f29-8d8b-1f36d9bc2d30)

thanks.

**Comment by 10cks** [comment]
Can you provide the complete code with the changes? I can try to analyze it.

**Comment by something-zero** [comment]
i just changed main function of CoercedPotato.cpp file.
file:
[CoercedPotato.zip](https://github.com/Prepouce/CoercedPotato/files/13223698/CoercedPotato.zip)

thanks

**Comment by 10cks** [comment]
The modified code you provided can be executed correctly in my case:
![image](https://github.com/Prepouce/CoercedPotato/assets/47177550/5e345d6a-2719-4419-9741-1d8c83ffb8bf)

**Comment by 10cks** [comment]
"CreateProcessAsUser() failed. Error:5" means ERROR_ACCESS_DENIED, you can try to set up exploitId parameters, test other functions whether there is the same problem.

**Comment by something-zero** [comment]
Are you running the code using the command line? Or do you click on the generated exe file?
My problem is that I can't run the file directly, but I can run it through CMD

**Comment by 10cks** [comment]
I successfully triggered this problem through your description, which is indeed strange, and I did not get a solution after many analyses, and now the people who are curious about this matter have changed from one person to two people.😄

**Comment by something-zero** [comment]
That's the point.
I ran the command: whoami /priv to make sure that the CreateProcessAsUser function has the necessary permissions. But still getting this error!😂

**Comment by 10cks** [comment]
I tested GodPotato in the same way and this did not happen, I wonder why this happened
![image](https://github.com/Prepouce/CoercedPotato/assets/47177550/f93c2a22-cd42-42c5-bd73-6356f2e0a5e6)

**Comment by something-zero** [comment]
yes **GodPotato** working correct.
but i want using c++

**Comment by 10cks** [comment]
yep, I was testing for potato rights, because earlier I felt that the problem was the difference between process and thread, because the simulation token needs to be inherited in the thread, looking at GodPotato's test results should have nothing to do with this.

**Comment by Hack0ura** [comment]
Hmm... That's strange :laughing:. My guess is the same as 10cks, it is probably a privilege inheritance problem when using GUI. I look for solving this problem but for now, I don't have any idea how to do it haha!
